addons/mail/static/src/models/rtc.js

```diff
@@ -312,7 +312,7 @@ registerModel({
             if (this.disconnectAudioMonitor) {
                 this.disconnectAudioMonitor();
             }
-            if (this.messaging.userSetting.usePushToTalk || !this.channel || !this.audioTrack) {
+            if (this.messaging.userSetting.isPushToTalkUsed || !this.channel || !this.audioTrack) {
```

Comment:
this rename is adding a lot of noise for not much, and the new name is not really better